openapi/rhose-api.yaml

```diff
@@ -877,6 +974,16 @@ components:
           type: string
         enabled:
           type: boolean
+    Error:
+      allOf:
+      - $ref: '#/components/schemas/ObjectReference'
+      - $ref: '#/components/schemas/Error'
```

Comment:
This looks strange. Error references itself? Might need fix

Reply:
fixed as https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/861/commits/49441d9e380db482ed116c27f5a0eba54c198e1d